docs/usage/available-methods-examples.md

```diff
@@ -34,7 +34,7 @@ $messageId = $response->getMessageId();
 See [sendChatAction](https://core.telegram.org/bots/api#sendchataction) docs for a list of supported actions and other info.
 
 ```php
-$telegram->sendChatAction('CHAT_ID', 'upload_photo');
+$telegram->sendChatAction(['chat_id' => 'CHAT_ID', 'action' => 'upload_photo']);
```

Comment:
For action, We also have a nice helper. We could use `Actions::UPLOAD_PHOTO` (maybe another example).